recipes/bokeh/run_test.py

```diff
@@ -0,0 +1,8 @@
+import sys
+import bokeh
+
+if sys.platform != 'win32':
+    bokeh.test()
```

Comment:
I think this is because until recently the tests would fail on Windows. This should not be necessary anymore though.